web-console/src/dialogs/snitch-dialog.tsx

```diff
@@ -27,6 +27,9 @@ import {
 import * as React from 'react';
 
 import { FormGroup, IconNames } from '../components/filler';
+import { localStorageGet, localStorageSet } from "../utils";
+
+const druidEditingAuthor = "DRUID_EDITING_AUTHOR";
```

Comment:
Would it make sense to consolidate keys that are being used for local storage... somewhere, like maybe export a const somewhere or something with all the keys and document what they do and what uses them, to make it easy to coordinate and re-use storage between views and make sure there are no conflicts? Or would this be too general? I noticed that there are dynamic storage key values I guess, from #7292 which uses the table name as the storage key so it's on the views to not step on each other or anything else.

Reply:
To add, not necessarily recommending this as part of this PR, just curious on best practices as much as anything else.

Reply:
I will create another PR to put all the table column selection local storage keys into one file

Reply:
LOL I meant capitalize `druidEditingAuthor`.
I think this line should be: `const DRUID_EDITING_AUTHOR = "druid-editing-author";`

Reply:
I'm actually doing this in the branch where I'm pulling all the local storage keys out into one file. Will update it when I'm making the next PR.